test/e2e/specs/site-editor/patterns.spec.js

```diff
@@ -125,18 +120,19 @@ test.describe.skip( 'Patterns', () => {
 				level: 2,
 			} )
 		).toBeVisible();
-		await expect( patterns.list.getByRole( 'listitem' ) ).toHaveCount( 1 );
+		await expect( patterns.item ).toHaveCount( 1 );
 		await expect(
-			patterns.list
-				.getByRole( 'heading', { name: 'My pattern' } )
-				.getByRole( 'button', { name: 'My pattern', exact: true } )
+			patterns.itemsList.getByText( 'My pattern', {
```

Comment:
Do we have a heading role for this text for accessibility?

Reply:
They are just divs.. I don't know if there is a better way to get them..

Reply:
I think there should be a heading role or something similar for the data view so that screen readers know what the grid is about. This can be a follow-up accessibility task for the data views though!